src/Elementor/Widgets/CommentForm/WidgetCommentForm.php

```diff
@@ -1496,37 +1660,245 @@ public function skinPrintTemplate( $content, $button ) {
      */
     protected function _content_template() {
 
-        echo ( new Template( __DIR__ . '/partials/editor-content.php' ) )->getRendered();
+        echo ( new Template( __DIR__ . '/partials/editor-content.php', [ 'id' => $this->get_id() ] ) )->getRendered();
```

Comment:
Can be simplyfied to
`new Template( __DIR__ . '/partials/editor-content.php', [ 'id' => $this->get_id() ] )->render();`